src/ast/node.cr

```diff
@@ -40,8 +40,8 @@ module Mint
         end
       end
 
+      property from : Int32
```

Comment:
This needed to change because it's used for the reordering.

Reply:
I'm not following why this needs to be declared earlier?

Reply:
Not earlier, it changed from a `getter` to a `property`.

Reply:
It's earlier because of my slight OCD of having longer lines at the top :joy: